File: 2daSemanaInformaticaYComputacionCUCEI/B/b_TeamWarriors.cpp
//Team Warriors Problem B
//No Send
#include <iostream>
#include <string.h>
using namespace std;

int main()
{
	int numFra,lim,cont=0;
	cin>>numFra;
	char frases[numFra][100];
	char faltantes[numFra][100];
	int pangram[numFra];
	for (int i = 0; i < numFra; ++i){
		cin>>frases[i];
	}
	for (int i = 0; i < numFra; ++i){
		for (int j= 0; j < strlen(frases[j]); ++j){
			if (!(frases[i][j]>='a' && frases[i][j]<='z')){
				faltantes[i][j]=frases[i][j];
				pangram[i]=0;
			}
			else{
				pangram[i]=1;
			}
		}
	}
	for (int i = 0; i < numFra; ++i){
		if (pangram[i]==0)
		{
			cout<<"missing "<<faltantes[i]<<endl;
		}
		else{
			cout<<"pangram"<<endl;
		}
	}

	return 0;
}

File: 2daSemanaInformaticaYComputacionCUCEI/B/b_TeamWarriors.py
#Team Warriors Problem B
#Yes
def main():
	acbecedario = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z']
	numFrase = int(input())
	frases = []
	faltantes = []

	for i in range (0,numFrase):
		frases.append(input())

	for x in frases:
		falta = []
		for letra in acbecedario:
			if letra not in x:
				if letra.upper() not in x:
					falta.append(letra)
		faltantes.append(falta)

	for f in faltantes:
		if len(f)==0:
			print("pangram")
		else:
			cadena = "".join(f)
			print("missing",cadena)



if __name__ == '__main__':
	main()